lib/qs/daemon.rb

```diff
@@ -22,15 +24,26 @@ def self.included(klass)
     module InstanceMethods
 
       attr_reader :daemon_data, :logger
+      attr_reader :signals_redis_key, :queue_redis_keys
 
       def initialize
         self.class.configuration.validate!
         @daemon_data = DaemonData.new(self.class.configuration.to_hash)
         @logger = @daemon_data.logger
 
+        @redis = HellaRedis::Connection.new(Qs.redis_config.merge({
```

Comment:
@jcredding do you need to require 'hella-redis' like you are requiring 'dat-worker-pool'?